templates/index.html

```diff
@@ -59,16 +59,16 @@
                 <!-- Wrapper for slides -->
                 <div class="carousel-inner">
                     <div class="item active">
-                        <img src="{{ url_for('static', filename='image/carousel1.jpg') }}">
+                        <img src="{{ url_for('static', filename='image/carousel1.jpg') }}" width="1920" height="700">
```

Comment:
width and height will be set automatically by css but if you set them responsive mode will be deactivated!!!
@itmard